book_maker/loader/epub_loader.py

```diff
@@ -120,7 +123,7 @@ def _process_paragraph(self, p, index, p_to_save_len):
                 new_p.string = self.translate_model.translate(p.text)
                 self.p_to_save.append(new_p.text)
 
-        p.insert_after(new_p)
+        self.helper.insert_with_style(p, new_p.string, self.translation_style)
```

Comment:
better:

```python
if elf.translation_style:
   do_something()
else:
    p.insert_after(new_p)
```
readable

Reply:
Is it better to use `self.helper.insert_with_style` here?  Because can make more judgments in insert_with_style. For example, if the translated result is the same as the original text, do not insert nodes,  to make epub_loader more concise. If no style is configured, there will be no other impact

Reply:
but `insert_with_style` let user's think its a `style` thing we use no style as default

Reply:
better:

```python
if elf.translation_style:
   self.helper.insert_with_style(p, new_p.string, self.translation_style)
else:
    p.insert_after(new_p)
```

Reply:
How about rename `insert_with_style` to `insert_trans`, style default == ""
```
        if translation_style != "":
            new_p["style"] = translation_style
```

Reply:
make sense